include/fi_rbuf.h

```diff
@@ -226,16 +227,15 @@ static inline int rbfdinit(struct ringbuffd *rbfd, size_t size)
 	if (ret < 0)
 		goto err1;
 
-	flags = fcntl(rbfd->fd[RB_READ_FD], F_GETFL, 0);
-	ret = fcntl(rbfd->fd[RB_READ_FD], F_SETFL, flags | O_NONBLOCK);
-	if (ret < 0)
+	ret = fi_fd_nonblock(rbfd->fd[RB_READ_FD]);
+	if (ret)
```

Comment:
We are in the process of using the ofi prefix for internal calls, but most still use fi.  I think the change here is fine, since it's just using an existing call.  I only asked that new calls use the newer prefix.  Updating the existing calls is really a separate issue.